README.md

```diff
@@ -199,6 +266,72 @@ mkShell {
 
 If you change any `pom.xml`, you must run `./nix/update-maven.sh`.
 
+# Quick Start for Ubuntu Bionic/Focal
+
+0.  (For WSL/virtual machine users only) be sure your Linux environment is
+    already installed by following
+    [the instructions above](#virtualized-system-configurations) and then
+    start your Linux environment or virtual machine. For WSL users, you can
+    enter the Linux command line environment by opening up `cmd.exe` and
+    typing the following command:
+
+    ```
+    wsl.exe
+    ```
+
+    For VirtualBox or virtual machine software users, you will need to open
+    the VirtualBox or virtual machine application and follow the graphical
```

Comment:
Likely some of this should go into `k-distribution/INSTALL.md` instead, and you should link there from here. Anything to do with getting K standing up on a given platform from our packaged releases should go into `INSTALL.md`. This guide should focus on (i) from-source builds for developers, and (ii) how developers can get examples running so they can begin working on K.